python/packages/ai/teams/ai/state/conversation_state.py

```diff
@@ -12,3 +13,5 @@ class ConversationState:
     inherit a new interface from this base interface to strongly type
     the applications conversation state
     """
+
+    __history__: List[str]
```

Comment:
this was updated to be a class `ConversationHistory` which I spoke about with the team.

Reply:
See task #444